tests/platform/test_reboot.py

```diff
@@ -139,6 +149,16 @@ def reboot_and_check(localhost, dut, interfaces, reboot_type=REBOOT_TYPE_COLD, r
         logging.info("Further checking skipped for %s test which intends to verify reboot-cause only".format(reboot_type))
         return
 
+    check_further_interfaces_and_services(dut, interfaces)
+
+
+def check_further_interfaces_and_services(dut, interfaces):
+    """
+    Perform the further check after reboot-cause, including transceiver status, interface status
+    @param localhost: The Localhost object.
+    @param dut: The AnsibleHost object of DUT.
+    @param interfaces: DUT's interfaces defined by minigraph
+    """
     logging.info("Wait some time for all the transceivers to be detected")
```

Comment:
create define for the time to wait and instead of wait sometime, print the actual time you are going to wait for,
BTW, how the time was calculated and why you chose this value?

Reply:
They are experimental values.